content/docs/custom-stacks/_index.md

```diff
@@ -0,0 +1,33 @@
++++
+title="Custom stacks"
+weight=4
+creatordisplayname = "Andrew Meyer"
+creatoremail = "ameyer@pivotal.io"
+lastmodifierdisplayname = "Andrew Meyer"
+lastmodifieremail = "ameyer@pivotal.io"
++++
+
+A [stack](/docs/using-pack/managing-stacks) provides the buildpack lifecycle with build- and run-time environments (given by a
+build image and a run image). If you're using the `pack` CLI, running `pack suggest-stacks` will display a list of recommended
+stacks that can be used when running `pack create-builder`, along with each stack's associated build and run images.
+
+To create a custom stack for use with `pack create-builder`, the following requirements must be met for the build and run images.
+
+### The stack must be given an identifier
+
+Each image must contain a label called `io.buildpacks.stack.id` with an identifier for the stack (e.g. `io.buildpacks.stacks.bionic`).
+The stack identifier implies ABI (application-binary interface) compatibility with other stacks of that same identifier. For instance,
+a custom stack may use `io.buildpacks.stacks.bionic` as its identifier so long as it will work with buildpacks that declare
+compatibility with the `io.buildpacks.stacks.bionic` stack.
+
+In addition to a label, the image must contain an environment variable called `CNB_STACK_ID` whose value is the stack identifier.
+
+### The stack must have a user and group
+
+The stack images must also set a user and group by means of the `CNB_USER_ID` and `CNB_GROUP_ID` environment variables.
+
+## Resources
+
+For a reference stack, see the [CNB Bionic stack repo](https://github.com/buildpack/stacks).
+
+For more details on stack requirements, see the [platform specification for stacks](https://github.com/buildpack/spec/blob/master/platform.md#stacks).
```

Comment:
Maybe add "technical" to this since this page already, and should, contain a good amount of detail.

> For more technical details on stacks, see the [platform specification for stacks](https://github.com/buildpack/spec/blob/master/platform.md#stacks).

Reply:
Great point 👍